flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -2528,7 +2528,14 @@ class TableEnvironmentTest {
     tableEnv.executeSql(sourceDDL)
 
     val expectedResult1 = util.Arrays.asList(
-      Row.of("c0", "CHAR(10)", Boolean.box(true), null, null, null, "this is the first column"),
+      Row.of(
+        "c0",
+        "CHAR(10)",
+        Boolean.box(true),
+        null,
+        null,
+        null,
+        "this is the first column, named 'c0'."),
```

Comment:
In my opinion, adding tests for `TableEnvironmentTest` and `SqlDdlToOperationConverterTest` doesn't make much sense, because they already worked before this PR.

Reply:
done